README.md

```diff
@@ -50,9 +50,16 @@ Edit `/_posts/2014-3-3-Hello-World.md` to publish your first blog post. This [Ma
 
 ## Local Development
 
-1. Install Jekyll and plug-ins in one fell swoop. `gem install github-pages` This mirrors the plug-ins used by GitHub Pages on your local machine including Jekyll, Sass, etc.
-2. Clone down your fork `git clone https://github.com/yourusername/yourusername.github.io.git`
-3. Serve the site and watch for markup/sass changes `jekyll serve`
+1. Clone down your fork `git clone https://github.com/yourusername/yourusername.github.io.git`
+2. Create the Gemfile below then `bundle` (note this does not get checked in):
```

Comment:
I'm guessing the Gemfile throws off Github Pages or something? Seems odd to ignore it and have everyone create it manually.

Reply:
I'm not sure why. We could try to create it and see what happens.

Reply:
Oh, nice. Yeah not a huge deal either way, just thinking that it might make setup a little easier

Reply:
@regexpressyourself -- Give that a go. I added the Gemfile and Gemfile.lock.
I also slashed the README down to to basics specific to us (mostly).

Reply:
👍  Looking good here. Thanks!